app/views/layouts/application.html.erb

```diff
@@ -15,6 +15,7 @@
     <nav>
       <li><%= link_to "Shelters Index", "/" %></li>
       <li><%= link_to "Pets Index", "/pets" %></li>
+      <li><%= link_to "Favorite Pets", "/favorites" %></li>
```

Comment:
95 lines of test code to test this one line

Reply:
I think @mikez321 said he was going to inquire about a better approach to this, but you are trooper for getting it done...

Reply:
Yeah, there has got to be a better way... props to you for this test!